Tools/scripts/generate_global_objects.py

```diff
@@ -293,6 +302,15 @@ def generate_runtime_init(identifiers, strings):
                 # since iter_files() ignores .h files.
                 printer.write(f'string = &_Py_ID({i});')
                 printer.write(f'PyUnicode_InternInPlace(&string);')
+        printer.write('')
+        printer.write('#ifdef Py_DEBUG')
+        printer.write("static inline void")
+        with printer.block("_PyStaticObjects_CheckRefcnt(void)"):
+            for i in immortal_objects:
+                with printer.block(f'if (Py_REFCNT({i}) < 999999999)', ';'):
```

Comment:
Hard-coding the specific refcount isn't ideal.  Consider adding a constant to Include/internal/pycore_object.h.

Reply:
Added `_PyObject_IMMORTAL_REFCNT` constant.